### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by nizarelouaer** [yadavaditya13/Liveness_Detector_using_CNN_Deep_Learning_OpenCV#1]
Hello, congratualations for the work. I tested the python script for liveness detection, however I am having a very low accuracy. Am I missing something  Thank you

**Comment by yadavaditya13** [comment]
Yep... The more the dataset you have.. the better it will work. I had about 14.5 k images as I wrote in ReadMe, but if you want accuracy just train the model more and more... you will be good to go then!...
